packages/atomic/src/index.html

```diff
@@ -7,26 +7,28 @@
 
     <script type="module" src="/build/atomic.esm.js"></script>
     <script nomodule src="/build/atomic.js"></script>
+    <link rel="stylesheet" href="/build/atomic.css" />
     <script>
       (async () => {
         await customElements.whenDefined('atomic-search-interface');
         const searchInterface = document.querySelector('#search');
 
         // Initialization
         /* searchInterface.initialize({
-          accessToken: 'my_token',
-          organizationId: 'my_org',
+          accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
+          organizationId: 'searchuisamples',
         }); */
 
-        searchInterface.afterInitialization(() => {
+        // Event emitted once the search interface is initialized correctly
+        searchInterface.addEventListener('ready', () => {
```

Comment:
As far as the API is concerned, instead of using an event like this, I think it would be more elegant if the `initialize` method returned a promise that is resolved when everything is `ready`.

So, for example: 

```
const searchInterface = document.querySelector('#search');

searchInterface.initialize({...)).then(()=> {
   const allStrings = searchInterface.i18n.getResourceBundle('en', 'translation');
})
```